hint.md

```diff
@@ -0,0 +1 @@
+I am a hint.
```

Comment:
There's a specific content block on GitBook for Hints. Can you figure it out from our docs and replace this paragraph with the proprietary markup?

Reply:
Thank you for letting me know! i added the correct format